conf/services.conf

```diff
@@ -0,0 +1,163 @@
+            object_id = ^ca_objects.object_id,
+            idno = ^ca_objects.idno,
+            Title = ^ca_objects.preferred_labels,
+            Author = ^ca_objects.Author,
+            Publisher = ^ca_objects.Publisher,
+            DateOfPublication = ^ca_objects.DateOfPublication,
+            PlaceOfPublication = ^ca_objects.PlaceOfPublication,
+            PublicationType = ^ca_objects.PublicationType,
+            Edition = ^ca_objects.Edition,
+            Size = ^ca_objects.Size,
+            Series = ^ca_objects.Series,
+            Subjects = ^ca_occurrences,
+            Collation = ^ca_objects.Collation,
+            ISBNISSN = ^ca_objects.ISBNISSN,
+            LibraryNumber = ^ca_objects.LibraryNumber,
+            Donated = ^ca_objects.Donated
+        }
+    },
+
+    # Photographs
+    photographs_search = {
+        type = search,
+        table = ca_objects,
+        restrictToTypes = [ photograph ],
+        checkAccess = [ 1 ],
+        content = {
+            object_id = ^ca_objects.object_id,
+            idno = ^ca_objects.idno,
+            Title = ^ca_objects.preferred_labels,
+            Creator = ^ca_objects.Creator,
+            DateOfCreation = ^ca_objects.DateOfCreation,
+            Publisher = ^ca_objects.Publisher,
+            DateOfPublication = ^ca_objects.DateOfPublication,
+            PlaceOfPublication = ^ca_objects.PlaceOfPublication,
+            Medium = ^ca_objects.Medium,
+            Subjects = ^ca_occurrences,
+        }
+    },
+    photographs_detail = {
+        type = detail,
+        table = ca_objects,
+        restrictToTypes = [ photograph ],
+        checkAccess = [ 1 ],
+        content = {
+            object_id = ^ca_objects.object_id,
+            idno = ^ca_objects.idno,
+            Title = ^ca_objects.preferred_labels,
+            Creator = ^ca_objects.Creator,
+            DateOfCreation = ^ca_objects.DateOfCreation,
+            Publisher = ^ca_objects.Publisher,
+            DateOfPublication = ^ca_objects.DateOfPublication,
+            PlaceOfPublication = ^ca_objects.PlaceOfPublication,
+            Medium = ^ca_objects.Medium,
+            PhysicalDescription = ^ca_objects.PhysicalDescription,
+            Subjects = ^ca_occurrences,
+            Summary = ^ca_objects.Summary,
+            Notes = ^ca_objects.Notes,
+            TermsOfUse = ^ca_objects.TermsOfUse
+        }
+    },
+
+    # Museum
+    museum_search = {
+        type = search,
+        table = ca_objects,
+        restrictToTypes = [ artwork, costume, museumArtefact ],
+        checkAccess = [ 1 ],
+        content = {
+            object_id = ^ca_objects.object_id,
+            idno = ^ca_objects.idno,
+            type = ^ca_objects.type_id,
+            ItemName = ^ca_objects.preferred_labels,
+            ItemDates = ^ca_objects.ItemDates,
+            Importance = ^ca_objects.Importance,
+            Subjects = ^ca_occurrences,
+            Classification = ^ca_objects.Classification
```

Comment:
Add:

```
             Description = ^ca_objects.Description,
             MakersMarks = ^ca_objects.MakersMarks,
             HistoricalDetails = ^ca_objects.HistoricalDetails,
             BibliographReferences = ^ca_objects.BibliographReferences,
             StatementOfSignificance = ^ca_objects.StatementOfSignificance
```

While these are text fields it doesn't hurt to search by them. Also wondering how we do `_fulltext`, but that can come later.